base/.mysql_profile

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [[ -z $MYSQL_HOST ]]; then
+  if [[ -z $STORAGE_PORT_3306_TCP_ADDR ]]; then  
+    echo "** ERROR: You need to link with a MySQL container as 'storage' or specify MYSQL_HOST env var."
+    echo "STORAGE_PORT_3306_TCP_ADDR (container link) or MYSQL_HOST should contain a MySQL hostname."
+    exit 1
+  fi
+  MYSQL_HOST=$STORAGE_PORT_3306_TCP_ADDR
+fi
+
+export MYSQL_HOST
```

Comment:
IOTW, I'm hesitant to copy/paste all available config, as it would likely cause config drift.. that said, maybe the implicit documentation could be worth it?